test/gtest/reduceextreme.hpp

```diff
@@ -324,3 +421,149 @@ struct ReduceExtremeTest : public ::testing::TestWithParam<ReduceExtremeTestCase
+        output_dev      = handle.Write(output.data);
+        output_grad_dev = handle.Write(output_grad.data);
+        input_grad_dev  = handle.Write(input_grad.data);
+        dim_dev         = handle.Write(dim.data);
+    }
+    void RunTest()
+    {
+        auto&& handle         = get_handle();
+        miopenStatus_t status = miopenStatusSuccess;
+
+        if(reduceExtremeOp == MIOPEN_REDUCE_EXTREME_AMIN ||
+           reduceExtremeOp == MIOPEN_REDUCE_EXTREME_AMAX)
+        {
+            cpu_aminmax_backward<T>(input, ref_input_grad, output, output_grad, count, dim);
+        }
+        // leave the rest of the cases for future backward implementation
+
+        if(reduceExtremeOp == MIOPEN_REDUCE_EXTREME_AMIN ||
+           reduceExtremeOp == MIOPEN_REDUCE_EXTREME_AMAX)
+        {
+            status = miopen::reduce::ReduceExtremeBackward(handle,
+                                                           input.desc,
+                                                           input_dev.get(),
+                                                           input_grad.desc,
+                                                           input_grad_dev.get(),
+                                                           output.desc,
+                                                           output_dev.get(),
+                                                           output.desc,
+                                                           output_grad_dev.get(),
+                                                           count.desc,
+                                                           count_dev.get(),
+                                                           dim.desc,
+                                                           dim_dev.get(),
+                                                           reduceExtremeOp);
+        }
+        // leave the rest of the cases for future backward implementation
+
+        ASSERT_EQ(status, miopenStatusSuccess);
+
+        input_grad.data = handle.Read<T>(input_grad_dev, input_grad.data.size());
+    }
+
+    void Verify()
+    {
+        double threshold = std::numeric_limits<T>::epsilon();
+        auto error       = miopen::rms_range(ref_input_grad, input_grad);
+
+        ASSERT_EQ(miopen::range_distance(ref_input_grad), miopen::range_distance(input_grad));
+        EXPECT_LT(error, threshold * 10) << "Error backward Output beyond 10xthreshold : " << error
+                                         << " Tolerance: " << threshold * 10;
+    }
+    ReduceExtremeTestCaseBwd reduceextreme_config;
+
+    tensor<T> input;
+    tensor<T> input_grad;
+    tensor<T> output;
+    tensor<T> output_grad;
+    tensor<int32_t> count;
+    tensor<int32_t> dim;
+
+    tensor<T> ref_input_grad;
+
+    miopen::Allocator::ManageDataPtr input_dev;
+    miopen::Allocator::ManageDataPtr input_grad_dev;
+    miopen::Allocator::ManageDataPtr output_dev;
+    miopen::Allocator::ManageDataPtr output_grad_dev;
```

Comment:
in `miopen`.h, you use naming scheme like xgrad, ygrad. At this file you change naming scheme, this maybe confusing

Reply:
I follow the template of this existing reduce op